UsabillaSampleApp/README.md

```diff
@@ -26,4 +35,124 @@ $ cordova-res
+
+### Add Platform
+
+- To add native platforms like android & ios , just run:
+
+```bash
+$ cordova platform add android ios
+```
+
+### Add Usabilla plugin
+
+- To add usabilla into cordova application , just run:
+
+```bash
+$ cordova plugin add usabilla-cordova --save
+```
+
+### Setup SDK
+
+-  Update Configurations to run sample app
+``` js
+<script type="text/javascript" src="js/config.js"></script>
+```
+``` js
+/// Usabilla Configuration
+const appId = 'YOUR_APP_ID_HERE';
+const formId = 'YOUR_FORM_ID_HERE';
+let event = 'YOUR_EVENT_TAG_HERE';
+const customVariable = {'YOUR_KEY_HERE': 'YOUR_VALUE_HERE'};
+```
+
+### Functions
+
+- Initialize the sdk and sets custom variables for targeting Campaigns.
+```
+initialize → String appId , Object customVariable
+```
+``` js
+    const customVariable = {'YOUR_KEY_HERE': 'YOUR_VALUE_HERE'};
+    Usabilla.initialize(function() {}, function (err) {}, appId, customVariable);
+```
+- Shows the Passive Form & Returns a JSON Object, contains result - rating, pageindex, sent flag.
+```
+loadFeedbackForm → String formId
+```
+``` js
+    Usabilla.loadFeedbackForm(function() {}, function (err) {}, formId);
+```
+- Load the Passive Form with current screen captured & Returns a JSON Object, contains result - rating, pageindex, sent flag.
+```
+loadFeedbackFormWithCurrentViewScreenshot → String formId
+```
+``` js
+    Usabilla.loadFeedbackFormWithCurrentViewScreenshot(function() {}, function (err) {}, formId);
+```
+- Load the Campaign & Returns a Map, contains result - rating, pageindex, sent flag.
+```
+sendEvent → String event
+```
+``` js
+    Usabilla.sendEvent(function() {}, function (err) {}, event);
+```
+- Reset Campaign, so can be triggered from fresh count.
+```
+resetCampaignData()
+```
+``` js
+    Usabilla.resetCampaignData(function() {}, function (err) {});
+```
+- Manually Dismiss the Forms / Campaign.
+```
+dismiss()
+```
+``` js
+    Usabilla.dismiss(function() {}, function (err) {});
+```
+- Sets data masking with default character / passed single character and based on masks rule.
+```
+setDataMasking → Array masks, String character
+```
+``` js
+    const masks = 'YOUR_Masking_Regex';
+    const maskCharacter = 'YOUR_Masking_Character';
+    Usabilla.setDataMasking(function() {}, function (err) {}, masks, maskCharacter);
+```
+- Returns masks array to check what is the defined rule.
+```
+getDefaultDataMasks()
+```
+``` js
+    Usabilla.getDefaultDataMasks(function() {}, function (err) {});
+```
+- Store forms in a cache (file storage), which can be used in a offline mode.
+```
+preloadFeedbackForms → List formIds
+```
+``` js
+    Usabilla.preloadFeedbackForms(function() {}, function (err) {}, formIds);
+```
+- Remove cached forms stored for offline use.
+```
+removeCachedForms()
+```
+``` js
+    Usabilla.removeCachedForms(function() {}, function (err) {});
+```
+- Remove cached forms stored for offline use.
```

Comment:
Duplicate line

probably we can say `Set debug mode to enable logging on console of Usabilla SDK errors and informations `

Reply:
Done